### PR TITLE
scheduler: logrus logger in reconciler

### PR DESCRIPTION
**Pull request by danilo-gemoli**
Use `logrus` logger in the reconciler as it is already being used in `plank`.

### CODE REVIEW

File: prow/scheduler/reconciler.go [hunk 1]
```diff
@@ -85,13 +89,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, fmt.Errorf("patch prowjob: %w", err)
 	}
 
-	logger.Info("Reconciliation completed")
+	log.Info("Reconciliation completed")
```
Comment (jmguzik) [inline prow/scheduler/reconciler.go:92]: is this log reallny needed? it holds little value

Comment (danilo-gemoli) [reply prow/scheduler/reconciler.go:92]: Maybe should I remove `Reconciliation started` as well?

Comment (jmguzik) [reply prow/scheduler/reconciler.go:92]: yeah, I think so

**Review by jmguzik — APPROVED** [review]

**Review by petr-muller — APPROVED** [review]

**Pull request merged**